src/cli/aspencli.py

```diff
@@ -145,6 +144,28 @@ def get_headers(self):
             headers["Cookie"] = f"_oauth2_proxy={os.getenv('OAUTH2_PROXY_COOKIE')}"
         return headers
 
+    def url_with_org(self, path):
```

Comment:
Kinda hacky (if you have better ideas that would be awesome!) patching in support for org id's in our paths in the CLI

Reply:
GraphQL, maybe. I don't have any better ideas at the moment